bitrise.yml

```diff
@@ -12,28 +12,44 @@ trigger_map:
 workflows:
   deploy:
     steps:
-    - activate-ssh-key@4:
-        run_if: '{{getenv "SSH_RSA_PRIVATE_KEY" | ne ""}}'
     - git-clone@6: {}
     - cache-pull@2: {}
+    - script@1:
+        run_if: '{{getenv "DEPLOY_VERSION_NUMBER" | ne ""}}'
+        title: Update version number
+        inputs:
+        - content: |-
+            #!/usr/bin/env bash
+            # fail if any commands fails
+            set -e
+
+            # Bump version
+            if [ $DEPLOY_VERSION_NUMBER != "" ]; then
+              sed -i '' "s/[0-9]*\.[0-9]*\.[0-9]*/$DEPLOY_VERSION_NUMBER/g" README.md *.podspec
```

Comment:
this regex is maybe a little too simple, and could lead to odd behaviour if some external dependencies are added to podspec...

Reply:
We don't want external dependencies so it's not really an issue.
But I fixed it.